### PR TITLE
docs: add guide to deploy k8s cluster in UCloud

### PR DESCRIPTION
**Pull request by uk8s**
Add guide to deploy k8s cluster in UCloud.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/setup/platform-setup/ucloud/index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Follow these instructions to prepare an
+[UCloud Container Service for Kubernetes(UK8S)](https://docs.ucloud.cn/uk8s/README)
+cluster for Istio.
+You can deploy a Kubernetes cluster to UCloud quickly and easily, which fully supports Istio.
+
+## Procedure
+
+1. Log on to the [UCloud UK8S console](https://console.ucloud.cn/uk8s/manage), and click **Create Cluster** button.
+
+1. Choose dedicated or managed cluster version. If dedicated cluster is chosen, 3 master nodes are required. If you choose managed version, the masters and core components like api-server, etcd, scheduler will be maintained by UCloud.
```
Comment (zirain) [inline content/en/docs/setup/platform-setup/ucloud/index.md:23]: Is there anything different between `dedicated` or `managed` when trying to install istio?

Comment (uk8s) [reply content/en/docs/setup/platform-setup/ucloud/index.md:23]: No，the  dedicated  version means user should manage the master node by themself. The managed cluster version means UCloud will manage the master node. But user has admin role in both cluster version. So there is no difference between dedicated and managed version when trying to install istio.

Comment (zirain) [reply content/en/docs/setup/platform-setup/ucloud/index.md:23]: I think it would be better if you can add some screenshots?

Comment (Xunzhuo) [reply content/en/docs/setup/platform-setup/ucloud/index.md:23]: +1

Comment (uk8s) [reply content/en/docs/setup/platform-setup/ucloud/index.md:23]: Screenshots has been added for deploy uk8s cluster.

**Comment by craigbox** [comment]
Hi uk8s,

The point of these pages isn't to list all the Kubernetes environments that Istio can be installed in, as the idea is that it should install in all of them out of the box.

Instead, it should point out anything special you need to do to make a cluster ready for running Istio.

This was important when there was special configuration required, but these days there doesn't seem to be much of that left.  I will be removing all these pages and moving them to an appendix later in the year.

I don't see anything in your example other than a "how to install" guide, so I'd rather not take it into the Istio docs at this time.

(Are you listed on our Ecosystem page?)

**Review by craigbox — COMMENTED** [review]
As there wasn't any further engagement to review, I'm going to close this PR.  Please feel free to re-open it if you are interested in bringing it up to date. 

/close

**Pull request closed without merging**